### ISSUE TITLE
[Bug]: MessageBar V9 example in documentation missing AriaLiveAnnouncer component / context.

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/fluentui#31504]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
N/A
```

### Are you reporting Accessibility issue?

yes

### Reproduction

React Components / v9

## Actual Behavior
in [V9 dismiss / add a message button](https://react.fluentui.dev/?path=/docs/components-messagebar--default#actions): selecting the button adds new messages but there isn't a `role = "alert"` / `"status"` nor an `aria-live` attribute. Screen reader doesn't seem to read anything either.

## Expected Behavior
Similar to how in [V8 Usage](https://developer.microsoft.com/en-us/fluentui#/controls/web/messagebar#usage),
various messageBar types, the right role and `aria-live` attribute is added.

Per Sarah, hat v9 example needs to be wrapped in the `AriaLiveAnnouncer` component/context to make the announce() calls work

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by ling1726** [comment]
The `AriaLiveAnnouncer` component needs to be created in userland since it's important not to have too many announcers in an application. Having one announcer per message bar would be detrimental to a11y experience

**Comment by ferBonnin** [comment]
@ling1726 if this is by design, should it be documented in the page that the example is not complete (to avoid a bad user experience for screen reader users), and the user will need to add the `AriaLiveAnnouncer` on their side?

I would argue this is similar to the **Usage** sample in V8 where the examples were hidden by default so the screen readers wouldn't immediately read the MessageBar without the user initiating the action.

**Comment by ling1726** [comment]
@ferBonnin  good point, reopening this issue for doc improvements

**Comment by miroslavstastny** [comment]
- add AriaLiveAnnouncer to the default stories wrapper
- add doc about aria live to all components which use it
